src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -244,7 +243,7 @@ export const WalletOptions: Definitions<WalletConfig> = {
     default: config =>
       config.deterministic
         ? DeterministicSeedPhrase
-        : randomAlphaNumericString(10, alea()),
+        : randomAlphaNumericString(10, new Alea(unseededRng)),
```

Comment:
I don't believe this is correct.

Here shows the prng of calling `(new Seedrandom())()`:

![image](https://user-images.githubusercontent.com/549323/109889444-40150980-7c3a-11eb-9eaa-793d45f80a77.png)

But using the code you provided I get a predictable prng from alea:

![image](https://user-images.githubusercontent.com/549323/109889516-5f139b80-7c3a-11eb-9d2c-fa7d1178665d.png)

Perhaps this should be:

```suggestion
        : randomAlphaNumericString(10, unseededRng),
```

I don't see a need to use Alea here, otherwise it would need to be `new Alea(unseededRng())`

Reply:
Ah, I good catch. I think I just missed _calling_ the seedrandom instance.